blocks/commerce-checkout/commerce-checkout.js

```diff
@@ -156,8 +157,8 @@ export default async function decorate(block) {
           <div class="checkout__block checkout__shipping-form"></div>
           <div class="checkout__block checkout__bill-to-shipping"></div>
           <div class="checkout__block checkout__delivery"></div>
-          <div class="checkout__block checkout__payment-methods"></div>
           <div class="checkout__block checkout__billing-form"></div>
+          <div class="checkout__block checkout__payment-methods"></div>
```

Comment:
Reordering the layout here so that it remains natural for "express checkout" payment methods like Apple Pay. Before this change, the billing address form appeared below the payment methods, requiring the user to scroll past the "Pay" button to fill in the billing address, to then scroll back up to the Apple Pay button to finish the checkout. 🙃 Putting the billing address form above the payment methods seems like a good compromise that works for both Credit Card and Apple Pay.